src/scores/emerging/emerging_impl.py

```diff
@@ -0,0 +1,478 @@
+"""
+This module contains methods which for emerging scores.
+"""
+
+from typing import Iterable, Optional
+
+import numpy as np
+import xarray as xr
+
+from scores.functions import apply_weights
+from scores.typing import FlexibleDimensionTypes, XarrayLike
+from scores.utils import gather_dimensions
+
+
+def risk_matrix_score(
```

Comment:
I would suggest putting a "*" after prob_threshold_dim. The purpose of this is to avoid people mucking up the order of their arguments. By forcing optional arguments to be keyword-only, certain programmer errors can be avoided.

Reply:
done as Tennessee suggested